testing/kubernetes/cleanup.sh

```diff
@@ -17,8 +17,6 @@
 set -x
 SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
-kubectl delete -f $SCRIPTDIR/gateway.yaml
-sleep 2
 kubectl delete -f $SCRIPTDIR/bookinfo.yaml
 sleep 2
 $SCRIPTDIR/run-controlplane-kubernetes.sh stop
```

Comment:
Same comment as the docker case. We can fold these two files cleanup.sh and run-controlplane-kubernetes.sh